x-pack/test/api_integration/apis/ml/results/max_anomaly_score.ts

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import expect from '@kbn/expect';
+import { Datafeed, Job } from '@kbn/ml-plugin/common/types/anomaly_detection_jobs';
+import { USER } from '../../../../functional/services/ml/security_common';
+import { FtrProviderContext } from '../../../ftr_provider_context';
+import { COMMON_REQUEST_HEADERS } from '../../../../functional/services/ml/common_api';
+
+export default ({ getService }: FtrProviderContext) => {
+  const esArchiver = getService('esArchiver');
+  const supertest = getService('supertestWithoutAuth');
+  const ml = getService('ml');
+
+  // @ts-expect-error not full interface
+  const JOB_CONFIG: Job = {
+    job_id: `fq_multi_1_ae`,
+    description: 'mean(responsetime) partition=airline on farequote dataset with 1h bucket span',
+    groups: ['farequote', 'automated', 'multi-metric'],
+    analysis_config: {
+      bucket_span: '1h',
+      influencers: ['airline'],
+      detectors: [
+        { function: 'mean', field_name: 'responsetime', partition_field_name: 'airline' },
+      ],
+    },
+    data_description: { time_field: '@timestamp' },
+    analysis_limits: { model_memory_limit: '20mb' },
+    model_plot_config: { enabled: false },
+  };
+
+  // @ts-expect-error not full interface
+  const DATAFEED_CONFIG: Datafeed = {
+    datafeed_id: 'datafeed-fq_multi_1_ae',
+    indices: ['ft_farequote'],
+    job_id: 'fq_multi_1_ae',
+    query: { bool: { must: [{ match_all: {} }] } },
+  };
+
+  async function createMockJobs() {
+    await ml.api.createAndRunAnomalyDetectionLookbackJob(JOB_CONFIG, DATAFEED_CONFIG);
+  }
+
+  async function runRequest(requestBody: object) {
+    const { body, status } = await supertest
+      .post(`/api/ml/results/max_anomaly_score`)
+      .auth(USER.ML_VIEWER, ml.securityCommon.getPasswordForUser(USER.ML_VIEWER))
+      .set(COMMON_REQUEST_HEADERS)
+      .send(requestBody);
+    return { body, status };
+  }
+
+  describe('GetAnomaliesTableData', function () {
```

Comment:
This suite name is the same as in the recently added `partition_fields_values` tests. Is that intended?

Reply:
no, good spot, fixed in https://github.com/elastic/kibana/pull/133776/commits/e7f4d950d018b5a9d92d6f9bb98592896b75f5df